NFTMint.py

```diff
@@ -0,0 +1,64 @@
+import getpass, os
+import sys
+from thirdweb import ThirdwebSDK
+from thirdweb.types.nft import NFTMetadataInput
+from eth_account import Account
+from dotenv import load_dotenv
+from web3 import Web3
+import io
+import warnings
+from IPython.display import display
+from PIL import Image
+from stability_sdk import client
+import stability_sdk.interfaces.gooseai.generation.generation_pb2 as generation
+import random
+import google.cloud.storage
+load_dotenv()
+def uploadFileToBucket(serviceAcctFile, bucket_name, blob_name, fileToUpload):
+    storage_client = google.cloud.storage.Client.from_service_account_json(serviceAcctFile)
+    bucket = storage_client.get_bucket(bucket_name)
+    blob = bucket.blob(blob_name)
+    blob.upload_from_filename(fileToUpload)
+RPC_URL = "https://rpc.testnet.fantom.network/"
+ETH_WALLET_PRIVATE_KEY = "0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
what

Reply:
I was getting saying an error that python was fed an expected amount of data. I solved it by presetting the variable byte size.